lib/marc_to_argot/macros/shared/notes.rb

```diff
@@ -273,6 +273,8 @@ def note_local_label(field)
         when '561'
           labels << 'Ownership history'
           labels << collect_subfield_values_by_code(field, '3').join(' ')
+        when '583'
+          labels << 'Action note'
```

Comment:
@kazymovae I think the $3 needs to be included as part of the label as it is for the 561 field above.